api/custom_auth/oauth/serializers.py

```diff
@@ -52,6 +52,7 @@ def create(self, validated_data):  # type: ignore[no-untyped-def]
             )
 
         user = self._get_user(user_info)
+        self.user = user
```

Comment:
```suggestion
```

We can access the user from the produced token. See comment below.

FWIW, in general it's best to avoid adding new attributes to objects mid-flight. If we need, we could update the class to be aware of this attribute and make it part of its mechanics.

Reply:
I think we still need to delete this line, right?